guides/common/modules/proc_configuring-repositories-deb.adoc

```diff
@@ -1,24 +1,26 @@
 [id="configuring-repositories-deb-{distribution-codename}"]
 
+:PuppetReleaseDeb: puppet7-release-{distribution-codename}.deb
+
 . Install the `ca-certificates` package:
 +
 [options="nowrap" subs="+quotes,attributes"]
 ----
 # {package-install} ca-certificates
```

Comment:
@evgeni `ca-certificates` is a recommended package for `wget`. Should we drop this instruction or keep it in to be explicit?

Reply:
I prefer being explicit. Also, nothing installs `wget`, so this line really should be `… wget ca-certificates` ;)